README.rst

```diff
@@ -48,6 +48,39 @@ All experiments are with ``mpi4py`` and will try to spawned workers depending on
 
 All figures can be generated using scripts in ``benchmarks``. Each script will generate and save the data to reproduce the figure. The figure can then be plotted by re-running the same script with the argument ``--plot``. The figures are saved in pdf in the ``benchmarks_results`` folder. The computation are cached with ``joblib`` to be robust to failures.
 
+.. note::
+
+   Open MPI tries to use all **up** network interfaces. This might cause the program to hang due to virtual network interfaces which could not actually be used to communicate with MPI processes. For more info `Open MPI FAQ <https://www.open-mpi.org/faq/?category=tcp#tcp-selection>`_.
+
+   In this case run the ``mpirun`` command:
+
+   - either spefifying usable interfaces using ``--mca btl_tcp_if_include`` parameter:
+
+   .. code-block:: bash
+
+	 $ mpirun -np 1 \
+		  --mca btl_tcp_if_include wlp2s0 \
+		  --hostfile hostfile \
+		  python -m mpi4py examples/plot_mandrill.py
```

Comment:
Dp you really need the `-m mpi4py`? I did not use it before but if this is necessary, ignore this omment.
```suggestion
		  python examples/plot_mandrill.py
```

Reply:
I preferred using that way considering the advice here: https://mpi4py.readthedocs.io/en/stable/mpi4py.run.html

Reply:
A very interesting read.
I think we should add the `MPI_COMM_WORLD.Abort()` in a`finalize` for dicodile in order to make it runnable without this `-m mpi4py` but in the mean time, let's keep it that way.